rosdep/base.yaml

```diff
@@ -3051,6 +3051,16 @@ libnl-dev:
   fedora: [libnl-devel]
   gentoo: [dev-libs/libnl]
   ubuntu: [libnl-dev]
+libnlopt-cxx-dev:
+  debian:
+    '*': [libnlopt-cxx-dev]
+    stretch: null
```

Comment:
It seems that `libnlopt-cxx-dev` will be available in the future in debian Stretch:
https://packages.debian.org/stretch-backports/libnlopt-cxx-dev

when that happens, this line can be removed.